### PR TITLE
[POM-298] Add debugging page

### PR DESCRIPTION
**Pull request by KPobeeNorris**
During the onboarding process we have received a number of queries from
users in regards to missing offenders, offenders with incorrect
information and so forth.  For example: for an offender to show up in
our service they need to be over 18, convicted, sentenced, etc.  At the
moment a dev needs to go and run a number of commands to try and find
out the various pieces of information and pass it along to whoever is on
duty.  Therefore, it was agreed that we should add a debugging page,
that anyone in the team can access, that provides a breakdown of the
individual offender to help us answer these types of queries.

### CODE REVIEW

**Comment by KPobeeNorris** [comment]
This PR relies on changes implemented in https://github.com/ministryofjustice/offender-management-allocation-manager/pull/736 and will need to be rebased after that PR has been approved/merged.

File: app/services/nomis/models/offender_base.rb [hunk 1]
```diff
@@ -44,6 +44,10 @@ def sentenced?
         SentenceTypeService.indeterminate_sentence?(imprisonment_status)
       end
 
+      def over_18?
+        age > 18
```
Comment (rossjones) [inline app/services/nomis/models/offender_base.rb]:
```suggestion
        age >= 18
```

File: app/controllers/debugging_controller.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  end
+
+private
+
+  def id
+    params[:offender_no]
+  end
+
+  def offender(offender_no)
+    return [] if offender_no.blank?
```
Comment (rossjones) [inline app/controllers/debugging_controller.rb]:
```suggestion
    return nil if offender_no.blank?
```

File: app/controllers/debugging_controller.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class DebuggingController < PrisonsApplicationController
+  def debugging
+    nomis_offender_id = id
+
+    @offender = offender(nomis_offender_id)
+    if @offender.present?
```
Comment (rossjones) [inline app/controllers/debugging_controller.rb]:
```suggestion
    unless @offender.blank?
```

**Pull request merged**